scripts/latency-test.sh

```diff
@@ -1,12 +1,10 @@
 #!/usr/bin/env bash
 # Runs a test to determine the latency of certain user actions
 
-set -eu
+set -euxo pipefail
+export RUST_LOG=info
 FM_FED_SIZE=${1:-4}
 ITERATIONS=${2:-10}
-if [ -z "${RUST_LOG:-}" ]; then
```

Comment:
No, no. Keep the ability to set custom `RUST_LOG`. It's very handy. Just change the default to `info`

Reply:
Good point. Updated all the shell-based tests to default to `info` but allow overriding ...